### DISCUSSION TITLE
Parallel display of ST7735

### DISCUSSION THREAD

**Discussion by 411956158** [Bodmer/TFT_eSPI#1965] [Compatible displays and setup files]
1. Hello, Dear Developer, I noticed that there is no parallel configuration available in setup.h of ST7735. Can it be replaced by other configuration files, or will it be supported in the near future? My equipment is ESP32S3, and I look forward to your reply.

2. IDE  **Arduino1.8.19**
3. TFT_eSPI library version  **2.4.72**
4. Board package version **2.0.4**
5. Procesor  **ESP32 S3**
6. TFT driver **ST7735S**
7. Interface **type Parallel**

**Comment by Bodmer** [comment]
A parallel interface may work but has not been tested.

With these small displays the SPI interface is simpler to implement and provides good performance. So I suggest using SPI if possible.
